pkg/scheduler/scheduler.go

```diff
@@ -88,7 +88,10 @@ func (pc *Scheduler) Run(stopCh <-chan struct{}) {
 	pc.cache.SetMetricsConf(pc.metricsConf)
 	pc.cache.Run(stopCh)
 	pc.cache.WaitForCacheSync(stopCh)
-	klog.V(2).Infof("Scheduler completes Initialization and start to run")
```

Comment:
We can add a log before  custom handler sync, so that user can check the latancy from log.

Reply:
Of course, I will add logs about the waiting cache initialized.